canonical-web-and-design/maas-ui
# Support logging in with macaroons

Done:
- Log in using macaroons. Fixes: https://github.com/canonical-web-and-design/maas-ui/issues/514.

QA:
- Configure a MAAS with external auth (see me for instructions).
- Check that you can log in with external auth configured and via regular api auth.
